<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title>Darya Vanichkina | PyConAU 2016 Presentation</title>
  <meta name="description" content="Darya Vanichkina's homepage.
">

  <link rel="shortcut icon" href="http://localhost:4000/assets/img/favicon.ico">

  <link rel="stylesheet" href="http://localhost:4000/assets/css/main.css">
  <link rel="canonical" href="http://localhost:4000/blog/2016/pycon2016/">
</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    
    <span class="site-title">
        
        <strong>Darya</strong> Vanichkina
    </span>
    

    <nav class="site-nav">
      <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

      <div class="trigger">
        <!-- About -->
        <a class="page-link" href="http://localhost:4000/">home</a>

        <!-- Blog -->
        <a class="page-link" href="http://localhost:4000/blog/">blog</a>

        <!-- Pages -->
        
          
            <a class="page-link" href="http://localhost:4000/404.html">404: Page not found</a>
          
        
          
        
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/biomart/">biomaRt</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/biomartseq/">Sequence with biomaRt</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/python/chromosomesize/">Make chromosome size file from reference fasta</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/contacts">contact</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/ct/">Pretty print to one column (equivalent of ls -1)</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/ngs/dgea/">Notes on differential gene expression analysis</a>
          
        
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/gitcontribute/">Contributing to other repositories</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/gitlog/">Customize how git log is displayed</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/grep/">grep</a>
          
        
          
        
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/junctions/">Simple awk scripts that extracts junctions from SAM</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/latex/">LaTeX notes</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/least1valuerow/">At least one value in row</a>
          
        
          
        
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/ngs/mats/">MATS</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/merge2df/">Merge more than 2 data frames</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/mixomics/">Mixomics PCA</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/modenotin/">Mode and not in</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/python/openwritelinebyline/">Open file and write line by line</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/orderdf/">Order data frame</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/outlierdetectionR/">Beautiful outlier test and visualization in R</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/overlapviolin/">Creating overlapping violin plots using ggplot2</a>
          
        
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/paste/">Paste tab-delimited text into data table in R</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/_pages/publications/">publications</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/sedwnewline/">Sed with newline and quotes</a>
          
        
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/ngs/star/">STAR</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/teaching/">teaching</a>
          
        
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/venn/">Venn</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/volcano/">Volcano</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/python/whatvariablesarehere/">All variables defined and their values</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wiki/">Bash</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/xlsx/">Export to xlsx</a>
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/xtable/">Pretty output with xtable</a>
          
        
          
        
          
            <a class="page-link" href="http://localhost:4000/wikipages/r/char2var/">Character to variable and vice versa</a>
          
        

        <!-- CV link -->
        <!-- <a class="page-link" href="http://localhost:4000/assets/pdf/CV.pdf">vitae</a> -->

      </div>
    </nav>

  </div>

</header>



    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">PyConAU 2016 Presentation</h1>
    <p class="post-meta">August 12, 2016</p>
  </header>

  <article class="post-content">
    <p>I presented a talk on <strong>Big data biology for pythonistas: getting in on the genomics revolution</strong> at PyCon Au 2016.</p>

<p>In the abstract, I promised the following:</p>

<p><small> <i> In 2001 Bill Clinton unveiled “the most important, most wondrous map ever produced by humankind” - the human genome. This monumental endeavour cost $3 billion, and took hundreds of scientists from all over the world 13 years. Today, a single person can generate such a map in ~2 days for $1000. This dramatic drop in cost means that we now have data for hundreds of thousands of people - and other species - from all corners of the globe, and cohorts are available for every major disease under the sun. Petabytes of new data are also being generated every week.</i></small></p>

<p><small> <i> Most of this data is publicly available, so anyone with an internet connection can try in silico biology from the comfort of their own home. In my talk, I’ll walk through what this data looks like, and how it’s analysed - with a special focus on where python fits into the workflow (;tldr the most interesting parts!). I will also highlight some common pitfalls software engineers and developers face when getting into this space. </i></small></p>

<p><small> <i> Finally, I’ll showcase several other facets of bioinformatics that sorely need contributions from good coders.
Genomics is rapidly entering the world of health care in both the public and private hospital sectors, and in direct-to-consumer genetic testing. Understanding this data, the challenges and limitations of its analytics will help us all make better-informed health and medical decisions, affecting our quality of life and those we love. </i></small></p>

<p>The slides for this:</p>

<iframe src="//www.slideshare.net/slideshow/embed_code/key/8n6RIYzhxBttVs" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen=""> </iframe>
<div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/DaryaVanichkina1/big-data-biology-for-pythonistas-getting-in-on-the-genomics-revolution" title="Big data biology for pythonistas: getting in on the genomics revolution" target="_blank">Big data biology for pythonistas: getting in on the genomics revolution</a> </strong> from <strong><a target="_blank" href="//www.slideshare.net/DaryaVanichkina1">Darya Vanichkina</a></strong> </div>

<p>See the full video on youtube <a href="https://youtu.be/fQ_vNHDNDCA">here</a>.</p>

<p>If you are trying to follow up on my talk and carry out an analysis of some data (or are looking for some data, or would like to know where to find the data from paper X), please leave a comment below, and I’ll do my best to answer your question and help! I don’t monitor the comments on youtube or slideshare, but I do get a ping if you leave a comment here.</p>

  </article>

  
    <div id="disqus_thread"></div>
    <script type="text/javascript">
      var disqus_shortname  = 'dvanic';
      var disqus_identifier = '/blog/2016/pycon2016';
      var disqus_title      = "PyConAU 2016 Presentation";
      (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
      })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  

</div>

      </div>
    </div>

    <footer>

  <div class="wrapper">
    &copy; Copyright 2018 Darya Vanichkina.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme.

    
  </div>

</footer>


    <!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<!-- Load Common JS -->
<script src="http://localhost:4000/assets/js/common.js"></script>


<!-- Load KaTeX -->
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css">
<script src="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.js"></script>
<script src="http://localhost:4000/assets/js/katex.js"></script>




<!-- Include custom icon fonts -->
<link rel="stylesheet" href="http://localhost:4000/assets/css/fontawesome-all.min.css">
<link rel="stylesheet" href="http://localhost:4000/assets/css/academicons.min.css">

<!-- Google Analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-XXXXXXXXX', 'auto');
ga('send', 'pageview');
</script>


  </body>

</html>
